app/views/sprint_feedbacks/show.tsx

```diff
@@ -0,0 +1,132 @@
+import { Stack } from '../../frontend/components/stack/stack';
+import { Card } from '../../frontend/components/card/card';
+import { Text } from '../../frontend/components/text/text';
+import { useModal } from '../../frontend/components/modal/modal';
+import { Button } from '../../frontend/components/button/button';
+
+export default function ({
+  data: { currentUser, feedback },
+}: PageProps<'sprint_feedbacks/show'>): JSX.Element {
+  const l = useFormatter();
+  const hourGoal = feedback.workingDayCount * 7.5;
+  const modal = useModal();
+
+  return (
+    <Layout user={currentUser} container>
+      <Stack>
+        <Stack line="mobile" justify="space-between">
+          <Text type="headline">{feedback.sprint.title}</Text>
+        </Stack>
+        <Card>
+          <Stack>
+            <Stack line="tablet" justify="space-between" size={2}>
+              <Text>
+                {l.dateRange(
+                  feedback.sprint.sprintFrom,
+                  feedback.sprint.sprintUntil
+                )}
+              </Text>
+              <Text>🔢 {feedback.finishedStorypoints}pts</Text>
+              <Text>
+                🔢 {l.singleDigitNumber(feedback.finishedStorypointsPerDay)}
+                pts/day
+              </Text>
+              <Text>⭐️ {feedback.retroRating}/5</Text>
+              <Text>💻 {feedback.workingDayCount} days</Text>
+            </Stack>
+            <Stack line="mobile" align="center" size={6}>
+              <img src={feedback.user.avatarUrl} style={{ maxWidth: '40px' }} />
+              <Text>{feedback.user.displayName}</Text>
+            </Stack>
+            <Stack line="desktop">
+              <Stack size={0}>
+                <Text>⏱ Sprint Overview</Text>
+                <Text>{l.percentage(feedback.trackedHours / hourGoal)}</Text>
+                <Text>
+                  {l.singleDigitNumber(
+                    feedback.trackedHours / feedback.workingDayCount
+                  )}{' '}
+                  hrs
+                </Text>
+                <Stack line="mobile">
+                  <Text>
+                    Missing:{' '}
+                    {l.singleDigitNumber(hourGoal - feedback.trackedHours)} hrs
+                  </Text>
+                  <Text>Billable {feedback.billableHours} hrs</Text>
+                </Stack>
+                <Stack line="mobile">
+                  <Text>Goal {l.singleDigitNumber(hourGoal)} hrs</Text>
+                  <Text>Tracked {feedback.trackedHours} hrs</Text>
+                </Stack>
+              </Stack>
+              <Stack size={0}>
+                <Text>⏱ Daily Overview</Text>
+              </Stack>
+              <Stack size={0}>
+                <Text>⭐ Retrospective</Text>
+                <Text>{feedback.retroRating}/5</Text>
+                <Text multiline>{feedback.retroText}</Text>
+                {feedback.permitEditRetroNotes && (
+                  <Button
+                    title="leave feedback"
```

Comment:
This now always says "leave feedback", even if I already gave some and want to edit it.

Also, when I first submit feedback, then click again to edit it, I get the empty form again. I need to do a full reload to have the form prefilled.

Reply:
I changed the modal helper to force bypass the cache to always display server data